features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```
@@ -99,7 +99,8 @@ repos:
 
 - name: raft
   path: raft
-  git: {<<: *git_defaults, repo: raft}
+  # TODO(jameslamb): revert this before merging
+  git: {<<: *git_defaults, repo: raft, upstream: jameslamb, tag: libraft-wheels}
```

Revert before merging:
```suggestion
  git: {<<: *git_defaults, repo: raft}
```